packages/cli/src/lib/cmds/init.ts

```diff
@@ -49,6 +50,12 @@ async function confirmTwaConfig(twaManifest: TwaManifest): Promise<TwaManifest>
       message: 'Name to be shown on the Android Launcher:',
       default: twaManifest.launcherName,
       validate: async (input): Promise<boolean> => notEmpty(input, 'Launcher name'),
+    }, {
+      name: 'display',
+      type: 'input',
```

Comment:
Since we have a small set of choices, can we use the `type: 'list'` prompt here? More info on https://github.com/SBoudrias/Inquirer.js#list---type-list